### ISSUE TITLE
Make tables more responsive

### ISSUE THREAD

**Issue by andrea-williams** [bcgov/wps#1718]
**As a** *Ops person/user of HFI Calc* **I want** *to be able to zoom out on the page* **So That** *I can see the entire table in one screen (even if it means that the font is small)*

**Additional Context**
- User Requested from sprint review for Sprint 47 (from Cariboo FC) (can be seen in Miro Sprint Review board)
- looks like right now we've set a fixed width and height for the table, so that if you zoom out now, the table width/height doesn't adjust and you just get a lot of wasted whitespace

**Acceptance Criteria**
- [ ] Given I'm logged in to HFI Calc, When I zoom out on the page in my browser, Then the table should still take up most of the width and height of the page, (and only the contents of the table should zoom out) so that I can look at the entire table view on one screen![Screen Shot 2022-02-14 at 11.46.17 AM.png](https://images.zenhubusercontent.com/5e331ea54a0d32f8650c9a25/006368df-31d1-42b6-a8ef-7bdbe5d15a3c)

**Definition of Done**
- [ ] Ready to Demo in Sprint Review
- [ ] Does what I have made have appropriate test coverage?
- [ ] Documentation and/or scientific documentation exists and can be found
- [ ] Peer Reviewed by 2 people on the team
- [ ] Manual testing of all PRs in Dev and Prod
- [ ] Merged

**Comment by tessmgood** [comment]
Note: relates to #1402, see CB's comments. This birthed #1410 (and epic #1662). Was parked for when we have time and have met deadlines / core MVP functionality is in. That said - is there a quick win here? Would be awesome. Thanks A for sliding this in.

**Comment by andrea-williams** [comment]
I don't think this would be toooooo time-intensive (famous last words, especially since it involves CSS). Part of me wants to see it prioritized since it was raised during a sprint review and I'd like to convey to our attendees that we're responsive to the issues we raise. 
But on the other hand, I know @conbrad has talked about soon-ish getting to a point where we can stop using the table and using a Grid layout instead, so maybe this issue is better left until then?

**Comment by conbrad** [comment]
I say we throw it all away when we get PDF download working :)

@tessmgood what do you think of the WeatherApp tables they have for their mountains of data? Maybe we should ask Dylan how they're creating them. Other than that, same concerns WRT to CSS tinkering which I'm terrible at.

**Comment by conbrad** [comment]
From a developer productivity and maintainability point of view, if we can leverage either the responsive MaterialUI grid (used in percentile calculator and fwi calculator (basIc), or we're comfortable with introducing the react-grid used in FWI calculator multi day those are the best options I know of.

**Comment by nmathava** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Wildfire-Predictive-Services-5e321393e038fba5bbe203b8/issues/bcgov/wps/1718?planning-poker) with ZenHub @andrea-williams @conbrad @Sybrand
